Documentation/observability/metrics.rst

```diff
@@ -38,7 +38,7 @@ Installation
 
 You can enable metrics for ``cilium-agent`` (including Envoy) with the Helm value
 ``prometheus.enabled=true``. To enable metrics for ``cilium-operator``,
-use ``operator.prometheus.enabled=true``.
+use ``operator.prometheus.enabled=true`` (enabled by default).
```

Comment:
Should this sentence now be flipped if the default is `true`? It's enabled by default, use ... = false to turn it off.